### PR TITLE
fix: url for print button to act like other print label buttons

### PR DESCRIPTION
**Pull request by xJoeyv**

### CODE REVIEW

File: includes/admin/views/html-order-track-trace-table.php [hunk 1]
```diff
@@ -61,8 +61,10 @@
           $order            = wc_get_order($order_id);
           $returnShipmentId = $order->get_meta(WCMYPA_Admin::META_RETURN_SHIPMENT_IDS);
 
+          $order_id = WPO\WC\MyParcel\Compatibility\Order::get_id($order);
```
Comment (EdieLemoine) [inline includes/admin/views/html-order-track-trace-table.php:64]: Camelcase, class graag importeren

File: includes/admin/views/html-order-track-trace-table.php [hunk 2]
```diff
@@ -61,8 +61,10 @@
...
+
           WCMYPA_Admin::renderAction(
-              admin_url("admin-ajax.php?action=$action&request=$getLabels&shipment_ids=$shipment_id&return_shipment_id=$returnShipmentId"),
+              admin_url("admin-ajax.php?action=$action&request=$getLabels&order_ids=$order"),
```
Comment (EdieLemoine) [inline includes/admin/views/html-order-track-trace-table.php:67]: Je stopt hier de hele order erin ipv het order id, heb je dit getest? Waarom vallen `shipment_ids` en `return_shipment_id` weg?

**Review by EdieLemoine — CHANGES_REQUESTED** [review]

File: includes/admin/views/html-order-track-trace-table.php (hunk 2 above)
Comment (xJoeyv) [reply includes/admin/views/html-order-track-trace-table.php:67]: De reden dat ik `shipment_ids` en `return_shipment_id` weg zijn uit de url is omdat nu de button exact dezelfde behavior vertoont als de andere print buttons.

De `$order` en `$order_id` is een foutje van mij.

**Review by EdieLemoine — DISMISSED** [review]

**Review by joerivanveen — DISMISSED** [review]

**Review by EdieLemoine — APPROVED** [review]

**Review by joerivanveen — APPROVED** [review]

**Review by joerivanveen — APPROVED** [review]

**Review by EdieLemoine — APPROVED** [review]
